### ISSUE TITLE
Problem with arrow keys

### ISSUE THREAD

**Issue by domi04** [holzschu/a-shell#472]
If I try to bring back the last command via the arrow keys, it doesn't work. Instead it types in OA if the up arrow is pressed, OC for the right, OD for the left and OB for the down key. I had this problem before, so I reinstalled the app and it was fixed for a short time. I guess it has something to do with ssh, because I recently used it quite often and then this problem occurred.

**Comment by holzschu** [comment]
You are right, I have the same problem and it's annoying. I don't think it comes from ssh, because I don't use it. Simply restarting the app is enough to bring the normal begaviour for the arrows back.

**Comment by WanderLanz** [comment]
As a sidenote, if you are somehow pressing `ctrl+[` that usually sends Escape on some limited keyboards and you might be accidentally invoking the ANSI escape. 

VT TTY stuff is just full of all sorts of non-standard standards 🙃, this is partially why doing anything terminal related with *just* wasm32-wasi is basically castrated to just basic ANSI escape over telnet, and why I am going insane trying to find out how to set raw mode since WASI is so unfathomably incomplete and only useful for WebDev because overinflated security egos think they can do better than the entirety of the standards committee before them.

Sorry for the rant
